Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/theme/Theme.kt

```diff
@@ -56,6 +56,14 @@ val DarkThemeColors = darkColors(
 val Colors.snackbarAction: Color
     get() = if (isLight) Purple300 else Purple700
 
+@Composable
```

Comment:
These color extensions aren't ideal, can they be derived from theme colors like `onSurface`? Perhaps push back on design?

Reply:
Will do. I'll change them when I hear back.